### ISSUE TITLE
JIT engine

### ISSUE THREAD

**Issue by walterpie** [ampersand-lang/etc-rs#14]
Add an optional JIT engine as a replacement to the bytecode interpreter for x86_64/amd64.

**Comment by shirshak55** [comment]
wow jit should be cool. But is there any runtime in lang anyway?

**Comment by walterpie** [comment]
Yes, the compiler executes a lot at compile-time. Also, you may optionally choose to interpret everything instead of compiling
